libs/ui/src/reboot_to_bios_button.tsx

```diff
@@ -22,12 +20,11 @@ export function RebootToBiosButton({ logger }: Props): JSX.Element {
       message: 'User trigged a reboot of the machine to BIOS screen…',
     });
     setIsRebooting(true);
-    // eslint-disable-next-line vx/gts-identifiers
     await window.kiosk.rebootToBios();
   }
 
   if (isRebooting) {
-    return <Modal content={<Loading>Rebooting…</Loading>} />;
+    return <Modal content={<Loading>Rebooting</Loading>} />;
```

Comment:
Unrelated to my changes! Noticed a double ellipsis here since the `Loading` component adds its own ellipsis. Also noticed some unneeded eslint disables

Reply:
my fault! Thanks for the fix.

Reply:
No prob!